README.md

```diff
@@ -63,4 +98,28 @@ class CustomCompletionFinder(argcomplete.CompletionFinder):
 completion_finder = cachedcomplete.cached_completion_finder(CustomCompletionFinder)()
 # Instead of argcomplete.autocomplete()
 completion_finder()
-```
\ No newline at end of file
+```
+
+### Specifying what files to track
+
+In order to specify which files to track changes in,
+add a comment with the wanted files and the prefix CACHEDCOMPLETE_HASH:
+
+```python
+# CACHEDCOMPLETE_HASH: file1.py
+```
+
+It also allows multiple files split to multiple comments, or within one comment.
+
+```python
+# CACHEDCOMPLETE_HASH: file1.py, file2.py
```

Comment:
No commas, separate with spaces - shell-like syntax

And maybe rename a file to be `.json` (or something else) instead to emphasize not only python files are supported?